doc/stdenv/meta.chapter.md

```diff
@@ -175,6 +175,37 @@ The NixOS tests are available as `nixosTests` in parameters of derivations. For
 
 NixOS tests run in a VM, so they are slower than regular package tests. For more information see [NixOS module tests](https://nixos.org/manual/nixos/stable/#sec-nixos-tests).
 
+Alternatively, you can specify other derivations as tests. You can make use of
+the optional parameter (here: `self`) to inject the correct package without
+relying on non-local definitions, even in the presence of `overrideAttrs`. This
+means `(mypkg.overrideAttrs f).tests` will behave as expected, as long as the
+definition of `tests` does not rely on the original `mypkg` or overrides it in
+all places.
+
+```nix
+# my-package/default.nix
+{ stdenv, callPackage, ... }:
```

Comment:
I think you should drop the `...` here and in the next file. It doesn't matter and has no usage. It would just confuse newcomers.

Reply:
Done.